### DISCUSSION TITLE
what is the default scheduler?

### DISCUSSION THREAD

**Discussion by yuvalkirstain** [Lightning-AI/pytorch-lightning#7314] [code help: NLP / ASR / TTS] [answered]
I see that there is an option to return just an optimizer in the `configure_optimizers` function. What will be the default scheduler in that case?
BTW I tried very hard to find the answer in the docs and could not find it..

**Comment by ananthsub** [comment, marked as answer]
There's no LR scheduler applied in this case: https://pytorch-lightning.readthedocs.io/en/latest/common/lightning_module.html#configure-optimizers

**Reply by yuvalkirstain** [reply]
so how is the learning rate determined?

**Reply by ananthsub** [reply]
The base learning rate is specified when constructing the optimizer: https://pytorch.org/docs/stable/optim.html
example: 
```
optimizer = optim.SGD(model.parameters(), lr=0.01, momentum=0.9)
```

**Reply by yuvalkirstain** [reply]
So it's just a constant learning rate? (thanks for the help BTW :))

**Reply by ananthsub** [reply]
Yup, a constant learning rate if no LR scheduler is provided
